bfvmm/src/exit_handler/test/test_exit_handler_intel_x64.cpp

```diff
@@ -2446,6 +2449,8 @@ exit_handler_intel_x64_ut::test_vmwrite_failure()
 
     RUN_UNITTEST_WITH_MOCKS(mocks, [&]
     {
-        EXPECT_EXCEPTION(eh->dispatch(), std::runtime_error);
+        auto f = [&] { eh->dispatch(); };
+        auto e = std::shared_ptr<std::exception>(new std::runtime_error("exit handler vmwrite failed"));
```

Comment:
I can change to make_shared. Could you give an example of what you would like to see?